server/db/User.js

```diff
@@ -7,11 +8,30 @@ class User extends Model {
       email: {
         type: STRING,
         validate: { isEmail: true },
-        unique: true
+        unique: true,
+        allowNull: false
       },
-      firstName: STRING,
-      lastName: STRING,
-      office: STRING
+      firstName: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+          len: [2, 100]
+        }
+      },
+      lastName: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+          len: [2, 100]
+        }
+      },
+      office: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+          is: office
+        }
+      }
```

Comment:
Also since you're tackling `User` model in this PR lets introduce `password` field and omit it in routes where needed.